<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #0f2027, stop:0.5 #203a43, stop:1 #2c5364);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1500</width>
     <height>1500</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QLabel" name="label_language">
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#label_language {
    color: white;
    font-weight: bold;
    font-size: 6pt;
}
</string>
      </property>
      <property name="text">
       <string>Languages</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="comboBox_language">
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox#comboBox_language {
    background-color: #ffffff;
    color: #2c5364;  /* Tema ile uyumlu koyu mavi */
    border: 2px solid #2c5364;
    border-radius: 5px;
    padding: 4px;
    min-width: 8px;
    font-weight: bold;
}
QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: #2c5364;
    selection-background-color: #203a43;
    selection-color: white;
}</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T√ºrk√ße</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dutch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fran√ßais</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>–†—É—Å—Å–∫–∏–π</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Deutsch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ÿßŸÑÿπÿ±ÿ®Ÿäÿ©</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ÁÆÄ‰Ωì‰∏≠Êñá</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Espa√±ol</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="3" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>18</number>
      </property>
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLabel" name="label_logo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/logo.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_search">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: white;
    color: #2c5364;           /* Tema rengine zƒ±t ve okunabilir */
    border: 2px solid #203a43;
    border-radius: 5px;
    padding: 5px;
    font-size: 12pt;
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_search">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #8f8f8f;
    color: white;
    border-radius: 8px;
    padding: 6px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/search.png</normaloff>../images/search.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_application">
        <property name="styleSheet">
         <string notr="true">QFrame {
    background-color: white;
    border-radius: 20px;
    padding: 5px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_application">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>üìÇ  Application Buttons</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_all_applications">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
             </property>
             <property name="text">
              <string>All Applications</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_planned_mentor_meetings">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
             </property>
             <property name="text">
              <string>Planned Mentor Meetings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_unscheduled_mentor_meetings">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}


</string>
             </property>
             <property name="text">
              <string>Unscheduled M. Meetings</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_vit">
        <property name="styleSheet">
         <string notr="true">QFrame {
    background-color: white;
    border-radius: 20px;
    padding: 5px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_vit_record">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>üìÇ  VIT Record Buttons</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_pre_vit_control">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
             </property>
             <property name="text">
              <string>Previous VIT Control</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_repeted_registeration">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
             </property>
             <property name="text">
              <string>Repeted Regitiration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_dif_registeration">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
             </property>
             <property name="text">
              <string>Diffrent Registiration</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_filter_options">
        <property name="styleSheet">
         <string notr="true">QFrame {
    background-color: white;
    border-radius: 20px;
    padding: 5px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>‚öôÔ∏è  Filter Options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_filter_application">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}
</string>
             </property>
             <property name="text">
              <string>Filter Application</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_BackMenu">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #8f8f8f;
    color: white;
    border-radius: 8px;
    padding: 6px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
           </property>
           <property name="text">
            <string>Back Menu</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../images/back.png</normaloff>../images/back.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>17</width>
             <height>17</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #8f8f8f;
    color: white;
    border-radius: 8px;
    padding: 6px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/logout.png</normaloff>../images/logout.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QTableWidget" name="tableWidget_application_dashboard">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>15</horstretch>
        <verstretch>15</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1400</width>
        <height>1000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
    background-color: #2c5364;
    color: white;
    padding: 4px;
    font-weight: bold;
    border: 1px solid #203a43;
}

QTableWidget {
    gridline-color: white;
    color: white;  /* ‚Üê Bu satƒ±r metinleri beyaz yapar */
}</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>155</number>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name Surname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mail</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telephone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Post Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Economical Situation</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_application_dashboard">
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-family: &quot;Georgia&quot;;
    font-size: 18pt;
    color: white;
}
</string>
      </property>
      <property name="text">
       <string>                                             Application Dashboard</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <spacer name="verticalSpacer_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
